test/signedTx/txGen.ts

```diff
@@ -46,21 +75,44 @@ const createInputTx = (unspents: any[], inputValue: number) => {
   return txInputBuilder.buildIncomplete();
 };
 
+function signInput(
+  txBuilder: bitcoin.TransactionBuilder,
+  index: number,
+  keys: bip32.BIP32Interface[],
+  unspent: IUnspent,
+) {
+  const nKeys = unspent.inputType === 'p2shP2pk' ? 1 : 2;
+  const prevOutScriptType = bitcoin.bitgo.outputScripts.isScriptType2Of3(unspent.inputType)
+    ? bitcoin.bitgo.outputScripts.scriptType2Of3AsPrevOutType(unspent.inputType)
+    : 'p2sh-p2pk';
+  keys.slice(0, nKeys).forEach((keyPair) =>
+    txBuilder.sign({
+      prevOutScriptType,
+      vin: index,
+      keyPair,
+      redeemScript: unspent.redeemScript,
+      witnessValue: (unspent.inputType === 'p2shP2pk' || unspent.inputType === 'p2sh')
+        ? undefined
+        : unspent.value,
+      witnessScript: unspent.witnessScript,
+    }),
+  );
+}
+
 class TxCombo {
-  public unspents: any[];
+  public unspents: IUnspent[];
   public inputTx: any;
 
   constructor(
     public keys: bip32.BIP32Interface[],
-    public inputTypes: TestUnspentType[],
+    public inputTypes: string[],
```

Comment:
`string` is a better type here (`TestUnspentType = string | UnspentTypeOpReturn`)